app/parse.py

```diff
@@ -14,5 +21,48 @@ class Course:
     course_type: CourseType
 
 
+def parse_course(course_soup: Tag,
+                 course_type: CourseType) -> Course:
+    return Course(
+        name=course_soup.select_one(".typography_landingH3__vTjok").text,
+        short_description=course_soup.select_one(
+            ".CourseCard_courseDescription__Unsqj"
+        ).text,
+        course_type=course_type,
+    )
+
+
+def get_driver() -> webdriver:
+    service = Service("/usr/local/bin/chromedriver")
+    options = Options()
+    options.add_argument("--headless")
+    driver = webdriver.Chrome(service=service, options=options)
+    driver.get(URL)
```

Comment:
move it from this function

Reply:
I don’t quite understand, do I need to move this to another function or remove it completely?

Reply:
As this function initializes the driver why do you get url? So let`s move `driver.get(URL)` to 47 line